test/blackbox-tests/test-cases/github3188/run.t

```diff
@@ -0,0 +1,5 @@
+  $ dune utop --display short testutop -- init_test.ml
```

Comment:
The `--display short` doesn't seem necessary here. We are trying to avoid its use when its not used to verify the correctness of the test

Reply:
Agreed that the display option isn't needed her. I've updated the pull request to remove that.

Reply:
Thanks